tck/mapping.ts

```diff
@@ -0,0 +1,33 @@
+import { sdk } from "./sdk_data";
+import { RpcMethodParams } from "./params/sdk";
+
+/**
+ * Very primitive catch-all mapping prototype
+ * @returns {Promise<*>}
+ * @param {Input} input
+ */
+export default async function mapMethods({
+    callClass,
+    methods,
+}: RpcMethodParams): Promise<string> {
+    const cl: any = (await import("@hashgraph/sdk"))[callClass];
+
+    let currentObject: any = new cl();
+    for (let { name, param } of methods) {
+        console.log(`.${name}(${param})`);
```

Comment:
You can remove the additional console.logs, they are going to flood the CLi and it's going to be harder when we use console.logs for debug purposes.